_posts/2015-08-04-creating-systems-with-pipelining.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Creating systems with pipelining"
+date: "2015-08-04"
```

Comment:
Please change the date to 2015-08-03. I will publish the blog post next Monday. Thank you.